evm/src/executor/inspector/debugger.rs

```diff
@@ -45,6 +45,12 @@ pub struct Debugger {
     ///
     /// For more information on gas blocks, see [current_gas_block].
     pub previous_gas_block: u64,
+    /// Call Trace tracker
+    pub call_trace_tracker: Rc<RefCell<Vec<usize>>>,
```

Comment:
looks like this is only used in `&mut self` so perhaps not necessary to put this in a rc refcell?

or perhaps that's the same value as in `Tracer::trace_stack`?

Reply:
this will be removed per @onbjerg's suggestion and we will deduce call trace index without the need for the refcell